srv/components/sspl/init.sls

```diff
@@ -19,7 +19,7 @@
 include:
   - components.sspl.prepare
   - components.sspl.install
-  - components.sspl.config
+  - components.sspl.config.init_mod
```

Comment:
Should be components.sspl.config, no?
Also, where is srv/components/sspl/config/init.sls that includes the 3 config sub-modules?
We need it until we make the switch to CES workflow

Reply:
similar to above implementation